src/test/queue.spec.ts

```diff
@@ -3,18 +3,11 @@ import * as sinon from "sinon";
 
 const { expect } = chai;
 
-import Queue = require("../queue");
+import Queue from "../queue";
 
 const TEST_ERROR = new Error("foobar");
 
 describe("Queue", () => {
-  it("should ignore non-number backoff", () => {
```

Comment:
This errors in TypeScript (because it's not a number) so I figure we probably don't need to test it anymore.

Reply:
Correct.